src/components/tiles/geometry/geometry-content.tsx

```diff
@@ -634,6 +636,29 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
     }
   }
 
+  private renderPolygonLabelDialog() {
+    const content = this.getContent();
+    const { board, showPolygonLabelDialog } = this.state;
+    if (board && showPolygonLabelDialog) {
+      const polygon = content.getOneSelectedPolygon(board);
+      if (!polygon) return;
+      const handleClose = () => this.setState({ showPolygonLabelDialog: false });
+      const handleAccept = (poly: JXG.Polygon, labelOption: ELabelOption, name: string) =>
+                            {
+                              this.handleLabelPolygon(poly, labelOption, name);
+                              handleClose();
+                            };
```

Comment:
This would normally be formatted something like:
```suggestion
      const handleAccept = (poly: JXG.Polygon, labelOption: ELabelOption, name: string) => {
        this.handleLabelPolygon(poly, labelOption, name);
        handleClose();
      };
```